src/github/interface.ts

```diff
@@ -47,6 +47,8 @@ export interface ISuggestedReviewer extends IAccount {
 export interface IMilestone {
 	title: string;
 	dueOn?: string | null;
+	createdAt?: string;

```

Comment:
shouldn't a milestone always have a `createdAt` and `id` field?

Reply:
I just wasn't getting them on all issues. Updated to include them.